packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const shortid = require('shortid');
```

Comment:
Sorry, I know there was some discussion about this before. I just want to understand why we do this.